spec/jobs/reindex_model_job_spec.rb

```diff
@@ -1,11 +1,26 @@
 # frozen_string_literal: true
 
 RSpec.describe HykuAddons::ReindexModelJob do
-  let(:account) { create(:account) }
-  let(:work) { create(:work) }
+  let(:account) { create(:account, cname: cname) }
+  let(:work) { create(:work, doi: []) }
+  let(:second_work) { create(:work, title: ["another work"], doi: []) }
```

Comment:
This isn't referenced anywhere in the test so we don't need to create it.

Reply:
@thenapking if you mean **second_work** it  was used in the test with issues that i spoke about in the stand up  i removed, so it ought to be off and will be removed.